site/content/docs/5.2/examples/form/index.html

```diff
@@ -0,0 +1,384 @@
+---
+layout: examples
+title: Form example
+extra_js:
+- src: "form.js"
+---
+
+<!-- Header -->
+<header class="sticky-top">
+  <nav class="navbar navbar-dark bg-dark navbar-expand-lg" aria-label="Global navigation - Standard example">
```

Comment:
The header should be the same as [in the DSM](https://system.design.orange.com/0c1af118d/p/20500e-form/b/16bb53)

![Screenshot 2022-11-10 at 08 35 21](https://user-images.githubusercontent.com/17381666/201028371-49f7a77f-053b-4ef6-b294-164f687247c1.png)

Reply:
Seen with @CyriaqueBillard, the page in the DSM will be updated to move the languages buttons to the suprabar since they are currently not allowed in the navbar.